iOS/Flows/Conversation/Conversation List/ConversationListViewController+Extensions.swift

```diff
@@ -59,21 +59,11 @@ extension ConversationListViewController {
         
         self.conversationListController
             .channelsChangesPublisher
-            .mainSink { [unowned self] changes in
-                let nonUpdateChanges = changes.filter { change in
-                    switch change {
-                    case .update:
-                        return false
-                    default:
-                        return true
-                    }
-                }
+            .mainSink { [unowned self] _ in
                 Task {
-                    await self.dataSource.update(with: nonUpdateChanges,
-                                                 conversationController: self.conversationListController,
-                                                 collectionView: self.collectionView)
-                }.add(to: self.taskPool)
```

Comment:
With the combination of the load more and placeholder conversation cells, we can't use the update array that stream gives us.

It's fine though. We can just pass in up to date list of conversations and have the datasource do a diff to check for changes.